packages/integration-test/test/lit-protocol.test.ts

```diff
@@ -0,0 +1,228 @@
+        },
+        payer: {
+          type: Types.Identity.TYPE.ETHEREUM_ADDRESS,
+          value: '0xb07D2398d2004378cad234DA0EF14f1c94A530e4',
+        },
+        timestamp: Utils.getCurrentTimestampInSecond(),
+      },
+      paymentNetwork: {
+        id: Types.Extension.PAYMENT_NETWORK_ID.ETH_FEE_PROXY_CONTRACT,
+        parameters: {
+          paymentNetworkName: 'sepolia',
+          paymentAddress: userWallet.address,
+          feeAddress: '0x0000000000000000000000000000000000000000',
+          feeAmount: '0',
+          tokenAddress: '0x0000000000000000000000000000000000000000',
+        },
+      },
+      contentData: {
+        meta: {
+          format: 'rnf_invoice',
+          version: '0.0.3',
+        },
+        creationDate: new Date().toISOString(),
+        invoiceNumber: 'INV-2023-001',
+        invoiceItems: [
+          {
+            name: 'Test Service',
+            quantity: 1,
+            unitPrice: ethers.utils.parseEther('0.1').toString(),
+            discount: '0',
+            tax: {
+              type: 'percentage',
+              amount: '0',
+            },
+            currency: 'ETH',
+          },
+        ],
+      },
+      signer: {
+        type: Types.Identity.TYPE.ETHEREUM_ADDRESS,
+        value: userWallet.address,
+      },
+    };
+
+    const encryptionParams = [
+      {
+        key: userWallet.address,
+        method: Types.Encryption.METHOD.KMS,
+      },
+    ];
+
+    const encryptedRequest = await requestNetwork._createEncryptedRequest(
+      requestParams as Types.ICreateRequestParameters,
+      encryptionParams,
+    );
+
+    await waitForConfirmation(encryptedRequest, 15, 2000);
+
+    const requestData = await encryptedRequest.getData();
+    expect(requestData).toBeDefined();
+    expect([Types.RequestLogic.STATE.CREATED, Types.RequestLogic.STATE.PENDING]).toContain(
+      requestData.state,
+    );
+
+    // Wait for any pending operations to complete
+    await new Promise((resolve) => setTimeout(resolve, 1000));
+  });
+
+  it('should handle encryption errors gracefully', async () => {
+    const invalidEncryptionParams = [
+      {
+        key: '',
+        method: Types.Encryption.METHOD.KMS,
+      },
+    ];
+
+    await expect(
+      litProvider.encrypt('test data', { encryptionParams: invalidEncryptionParams }),
+    ).rejects.toThrow(/invalid.*key/i);
+  });
+
+  it('should handle decryption errors gracefully', async () => {
+    const invalidEncryptedData = {
+      ciphertext: 'invalid-ciphertext',
+      dataToEncryptHash: 'invalid-hash',
+    };
+
+    await expect(
+      litProvider.decrypt(invalidEncryptedData, {
+        encryptionParams: [
+          {
+            key: userWallet.address,
+            method: Types.Encryption.METHOD.KMS,
+          },
+        ],
+      }),
+    ).rejects.toThrow();
+
+    // Wait for any pending operations to complete
+    await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
Same as the previous comment. Please remove unless this is strictly necessary.

Reply:
Let me test

Reply:
yes, removing it works